includes/cli/class-co-authors-plus.php

```diff
@@ -79,20 +79,6 @@ public static function unschedule_author_term_backfill() {
 	/**
 	 * Migrate Co-Authors Plus guest authors to regular users.
 	 *
-	 * ## OPTIONS
-	 *
-	 * [--live]
-	 * : Run the command in live mode, updating the subscriptions.
-	 *
-	 * [--verbose]
-	 * : Produce more output.
-	 *
-	 * [--user_logins]
-	 * : Comma-separated list of user logins. If provided, only WP Users with these logins will be processed.
-	 *
-	 * [--guest_author_ids]
-	 * : Comma-separated list of Guest Author IDs. If provided, only Gues Authors with these IDs will be processed.
-	 *
```

Comment:
Removing inline documentation makes the `--help` flag (`wp newspack migrate-co-authors-guest-authors --help`) unhelpful:

| Before | After | 
|--------------|-----------|
| <img width="922" alt="image" src="https://github.com/user-attachments/assets/0b7b0b9d-3a77-429d-9ec3-000b97a623e0"> | <img width="542" alt="image" src="https://github.com/user-attachments/assets/ad28ce84-59d7-482a-918e-657ed0794dee"> |

I'm not a fan of decoupling the docs from the code. When all commands are properly documented inline, WP CLI provides an excellent summary by itself (`wp newspack --help`), so there should be no need for a README:

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/bc251155-f911-4bbc-8147-d545be820822">

Reply:
@adekbadek a78f11b standardizes the use of docblocks instead of a separate README.md file.